### ISSUE TITLE
Can't find information of icons on the dashboard

### ISSUE THREAD

**Issue by luckyzor** [rstrouse/ESPSomfy-RTS#359]
### Hardware

ESP32

### Firmware version

2.4.2

### Application version

2.4.2

### What happened? What did you expect to happen?

I can't find what are the icons on the shades I have:
![image](https://github.com/rstrouse/ESPSomfy-RTS/assets/31155573/0cfdda2d-f172-40de-a186-4b2b394996e5)

### How to reproduce it (step by step)

```bash
go to the esp32 ip and check the dashboard
```

**Comment by rstrouse** [comment]
Those icons are windy and sunny condition icons.  At some point you likely sent sunny and windy conditions using the virtual remote.

Open up the virtual remote tab, select an afflicted shade from the dropdown and send a sensor command without any of the checkboxes checked.
![image](https://github.com/rstrouse/ESPSomfy-RTS/assets/47839015/0f06fe1d-5205-436b-b2a6-1478c2d001ca)

**Comment by rstrouse** [comment]
@luckyzor did performing the above operation clear the windy and sunny flags from your shutters?

**Comment by luckyzor** [comment]
Yes, sorry for the late reply, I went on vacations and totally forgot to test it.
It's cleared now, no more weird icons 😅
Tks
